pkg/controller/association/controller/beat_es.go

```diff
@@ -88,23 +88,23 @@ func getBeatRoles(assoc commonv1.Associated) (string, error) {
 			"kibana_admin",
 			"ingest_admin",
 			"beats_admin",
-			"remote_monitoring_user",
+			"remote_monitoring_agent",
```

Comment:
Should this bugfix be a separate PR?

Reply:
The rest of the PR validates the bugfix, no other test is utilizing that role which is why it was only discovered now.